projects/plugins/boost/app/modules/optimizations/minify/Minify.php

```diff
@@ -2,16 +2,13 @@
 
 namespace Automattic\Jetpack_Boost\Modules\Optimizations\Minify;
 
-use Automattic\Jetpack_Boost\Contracts\Pluggable;
+class Minify {
+	private static $called = false; // This class is called multiple times, so we need to make sure we only run some things once.
 
-// Allow overriding WordPress globals, as that is necessary to taking over script output.
-// phpcs:disable WordPress.WP.GlobalVariablesOverride.Prohibited
-
-class Minify implements Pluggable {
 	// @todo - handle PHP constants.
 
 	public function setup() {
-		require __DIR__ . '/functions-helpers.php';
+		require_once __DIR__ . '/functions-helpers.php';
 
 		// TODO: Make concat URL dir configurable
```

Comment:
`TODOs` should be turned into tickets

Reply:
Opened an issue listing all todos https://github.com/Automattic/jetpack/issues/30080.